features/polyExplorer/src/screens/dataExploration/dataExploration.css

```diff
@@ -157,7 +157,7 @@
 }
 
 .static-content-jurisdictions .info-button {
-    bottom: 200px;
+    bottom: 130px;
```

Comment:
Any specific reason for this number?

Reply:
Suppose the button was too high - I guess the number is either eye balled or measured on Figma :D

Reply:
Exactly, the button was too high (and yes, it was completely eye balled. I'm testing it a bit better and I think I have to change it again anyway...).